src/components/Select/Select.scss

```diff
@@ -17,32 +17,30 @@ $stacking-order: (
 
 .disabled {
   .Content {
-    color: color('ink', 'lightest');
+    color: var(--p-text-disabled, color('ink', 'lightest'));
   }
 
   .InlineLabel {
     color: inherit;
   }
 
   .Icon {
-    opacity: 0.4;
-  }
-
-  .Backdrop {
-    @include control-backdrop(disabled);
-  }
-}
-
-.error {
-  .Backdrop {
-    @include control-backdrop(error);
-  }
-
-  // Need to override the higher specificity of the sibling selector
-  // so that errors still have red borders.
-  // stylelint-disable-next-line selector-max-specificity
-  .Input:focus ~ .Backdrop {
-    @include control-backdrop(focused-error);
+    opacity: var(--override-one, 0.4);
+    @include recolor-icon(var(--p-icon-disabled, color('ink', 'lighter')));
+  }
+
+  &.globalTheming {
+    .Backdrop {
+      border-color: var(--p-border-secondary-disabled);
```

Comment:
Both secondary and onSurface variants are used for borders around inputs. This should be consistent, one or the other. I lean towards onSurface.

Reply:
/cc @danrosenthal

Reply:
All interactive elements should use the secondary borders, that creates a separation between the decorative (on surface) borders, and functional borders so we can ensure color contrast on interactive elements. We should fix any uses of on surface borders in interactive elements